pkg/config/example_test.go

```diff
@@ -345,6 +346,114 @@ func generateCert() *x509.Certificate {
 	}
 }
 
+const (
```

Comment:
Still not a fan of certificate fixtures without information on generating them. I thought we got rid of a fair bit of this recently with changes that @wlahti made.

Reply:
The commit I'm working on to generate the certs/CRL will handle this.